README.md

```diff
@@ -80,7 +81,8 @@ val registry: HttpMetricsRegistry = ... // concrete registry implementation
 
 val route: Route = ... // your route
 
-Http().newMeteredServerAt("localhost", 8080, registry).bindFlow(route)
+val routesWithMetrics = HttpMetricsRoute(routes).recordMetrics(registry)
+Http().bindAndHandle(routesWithMetrics, "localhost", 8080)
```

Comment:
This is the deprecated `akka-http` API. if you are still using akka 10.1, Use version [`v1.1.1`](https://github.com/RustedBones/akka-http-metrics/tree/v1.1.1) of the library

Reply:
In v1.1.1 version also, I was not able to get the function newMeteredServerAt()

Reply:
- In `v1.1.1` use 
```scala
Http().bindAndHandle(route.recordMetrics(registry), "localhost", 8080)
```
If you navigate to the [README.md for this release](https://github.com/RustedBones/akka-http-metrics/tree/v1.1.1) the doc is correct

- In `v1.2.0` use
```scala
Http().newMeteredServerAt("localhost", 8080, registry).bindFlow(route)
```
The README.md on the project page is from the `master` branch. It documents the latest (and potentially unreleased) version of the lib

Reply:
One side note, for version `v.1.2.0` to get access to `newMeteredServerAt` you must not forget to import implicits like 
```scala
import fr.davit.akka.http.metrics.core.HttpMetrics._
```